app/controllers/manage_proposal_controller.rb

```diff
@@ -123,7 +128,13 @@ def publish_create_or_update_proposal(proposal, provider)
 
     if cmr_response.success?
       flash[:success] = I18n.t('controllers.manage_proposals.publish.flash.create.success')
-      Rails.logger.info("Audit Log: Draft #{proposal['entry_title']} was published by #{current_user.urs_uid} in provider: #{provider} by proposal with short name: #{proposal['short_name']} and id: #{proposal['id']}")
+      Rails.logger.info("Audit Log: Proposal #{proposal['entry_title']} was published by #{current_user.urs_uid} in provider: #{provider} by proposal with short name: #{proposal['short_name']} and id: #{proposal['id']}")
+
+      dmmt_response = cmr_client.dmmt_update_proposal_status({ 'draft_type': proposal['draft_type'], 'id': proposal['id'] }, token, request)
+      unless dmmt_response.success?
+        flash[:error] = I18n.t('controllers.manage_proposals.publish.flash.update_proposal_status.error')
+        Rails.logger.info("Audit Log: dMMT did not successfully transition #{proposal['draft_type']} with id #{proposal['id']} to 'done' status from 'approved'.")
```

Comment:
also needs to state it is a Proposal that did not transition

Reply:
As we have only implemented collection proposals, proposal['draft_type'] will currently be something like CollectionDraftProposal.  Would you like it explicitly stated?